resolwe_bio/processes/reads_processing/cutadapt.yml

```diff
@@ -405,12 +405,10 @@
       # Run Cutadapt on both reads
       {% if minlen %}
         if [ -f ${NAME1}_trimmed.fastq.gz ]; then
-         zcat ${NAME1}_trimmed.fastq.gz > in_fw_reads.fastq
-         gzip in_fw_reads.fastq
+         cat ${NAME1}_trimmed.fastq.gz > in_fw_reads.fastq.gz
```

Comment:
Reads are often large, so it would make sense to just move the file `mv ${NAME1}_trimmed.fastq.gz in_fw_reads.fastq.gz` instead of copying. The same goes for the reverse reads.